ddtrace/tracer/tracer.go

```diff
@@ -269,6 +273,11 @@ func newTracer(opts ...StartOption) *tracer {
 			t.reportRuntimeMetrics(defaultMetricsReportInterval)
 		}()
 	}
+	if c.debugAbandonedSpans {
+		log.Warn("Abandoned spans logs enabled.")
```

Comment:
Should this be a warning? Perhaps info or debug is enough?

Reply:
Changed to Info, as other features use it too to inform when they are activated.